flybywiresim/a32nx
# feat: React Chrono, BAT indicator, rudder trim indicator

Fixes #3237
Fixes #3238
Replaces #3469

## Summary of Changes

- [x] Reimplement BAT indicator in react
- [x] Reimplement RTPI in react
- [x] Reimplement Chrono in react
   - [x] Stopwatch
   - [x] Clock/Date - **Note: switching from GPS time to internal time, and changing the internal time is not implemented yet.**
   - [x] Elapsed Time
      - [x] XML - **Note: the elapsed time knob should be springloaded in the 3rd position (reset). This is not yet implemented. If someone has an idea of how to do this, feel free to make a suggestion.**
* The visuals should be pretty much identical. 

## Screenshots (if necessary)



## References

* Chrono: https://youtu.be/xczl6YiLRl0
* RTPI: https://www.youtube.com/watch?v=hRWg4g3UiHo




## Additional context



Discord username (if different from GitHub):

## Testing instructions

* Make sure the chrono and RTPI behave according to the references provided. (Except the issue described above)
* Make sure the BAT indicator behaves properly.


## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page
